app/components/layouts/improvedHeader.tsx

```diff
@@ -226,7 +226,10 @@ class ImprovedHeader extends React.PureComponent<HeaderProps, HeaderStates> {
           aria-label="Scinapse small header logo"
         >
           <Icon icon="SCINAPSE_LOGO_SMALL" />
-          <div className={styles.instituteName}>{currentUserState.ipInstitute.nameAbbrev}</div>
+          <div className={styles.instituteLogo}>
+            <div className={styles.serviceName}>scinapse</div>
+            <div className={styles.instituteName}>{currentUserState.ipInstitute.nameAbbrev}</div>
```

Comment:
Is the `nameAbbrev` field always not null?
If it doesn't matter, LGTM!

Reply:
That is always not null. (There is no data outside of `Pluto Institute info` right now.)
and, `nameAbbrev` is the data we created. So, it's OK